src/vector/platform/ElectronPlatform.js

```diff
@@ -21,7 +21,7 @@ import VectorBasePlatform, {updateCheckStatusEnum} from './VectorBasePlatform';
 import dis from 'matrix-react-sdk/lib/dispatcher';
 import { _t } from 'matrix-react-sdk/lib/languageHandler';
 import Promise from 'bluebird';
-import {remote, ipcRenderer} from 'electron';
+import electron, {remote, ipcRenderer} from 'electron';
```

Comment:
why not just import desktopCapturer into the destructured imports instead of importing the whole of electron
you're already "slighltly-modifying" the copied code

Reply:
Done.